### DISCUSSION TITLE
Ubuntu Upgrade with Apache2 and SnipeIT

### DISCUSSION THREAD

**Discussion by chrish112** [snipe/snipe-it#14985] [Q&A]
Hi Everyone,
I recently upgraded from Ubuntu 20 to 22 on a backup and I noticed the website is now displaying the below picture. I didn't turn on a maintenance mode. I have a similar issue with another website not loading the PHP files, assuming it's Apache loading the PHP files. Anyone know how to get this back up?
![Screenshot 2024-06-27 110535](https://github.com/snipe/snipe-it/assets/160979202/1cc6648a-8229-4d25-ad11-5b37c2f91321)

**Comment by mauro-miatello** [comment]
I have had the same issue upgrading Snipe from v6 to v7, in my case I solved with
`sudo a2enmod php8.1 && sudo systemctl restart apache2`
